src/lib/api.js

```diff
@@ -31,7 +31,7 @@ export function disableGlobalGraphQLErrorHandling() {
 	globalGraphQLErrorHandlingEnabled = false;
 }
 
-export default async function API(): Promise<ApolloClient> {
+export default async function API( exitOnError = true ): Promise<ApolloClient> {
```

Comment:
This would be more readable (in the usages) if we transform it into an options object.
```javascript
const api = await API( { exitOnError: false } );
```
it's much easier to understand than:
```javascript
const api = await API( false );
```

```suggestion
export default async function API( { exitOnError = true } = {} ): Promise<ApolloClient> {
```

Reply:
Sounds good, I'll change both! Thanks!